### PR TITLE
CAM-12345 Adjust misc examples

### PR DESCRIPTION
**Pull request by yT0n1**
Related to CAM-12345

### CODE REVIEW

File: multi-tenancy/schema-isolation/standalone_test.xml [hunk 1]
```diff
@@ -148,8 +150,64 @@
...
+									
+									   
+																   
+								 
+									   
+									 
+									  
+								   
+						   
+
```
Comment (tasso94) [inline multi-tenancy/schema-isolation/standalone_test.xml:208]:
```suggestion
```
Can we remove these empty lines?

File: spin/dataformat-configuration-global/README.md [hunk 2]
```diff
@@ -113,6 +113,5 @@ Where to place the data format configuration JAR on which application server:
 
 | Application Server           | Configuration                                                           |
 | ---------------------------- | ----------------------------------------------------------------------- |
-| Tomcat                       | Put the JAR and its transitive dependencies into `${TOMCAT_HOME}/lib`.  |
```
Comment (tasso94) [inline spin/dataformat-configuration-global/README.md:116]: Why was Tomcat removed here?

File: multi-tenancy/schema-isolation/standalone.xml [hunk 3]
```diff
@@ -148,8 +150,64 @@
...
+									
+									   
+																   
+								 
+									   
+									 
+									  
+								   
+						   
+
```
Comment (tasso94) [inline multi-tenancy/schema-isolation/standalone.xml:208]:
```suggestion
```
Can we remove these empty lines?

**Review by tasso94 — CHANGES_REQUESTED** [review]
Additionally, to the review hints below, please test the example `multi-tenancy/schema-isolation` and update the `pom.xml` to version `7.14.0`.

File: spin/dataformat-configuration-global/README.md (hunk 2 above)
Comment (mboskamp) [reply spin/dataformat-configuration-global/README.md:116]: The example does not work with Tomcat and is not easy to fix. The pom specifies the `camunda-spin-dataformat-json-jackson` dependency, but tomcat provides `camunda-spin-dataformat-all` which shades the previous one.

Comment (mboskamp) [reply spin/dataformat-configuration-global/README.md:116]: Sorry, should have clarified before assigning to you.

Comment (tasso94) [reply spin/dataformat-configuration-global/README.md:116]: This makes sense. Let's keep the change 👍

**Comment by mboskamp** [comment]
@tasso94, `picketbox` is only available on an early access repo and not required for our usecase. See [CAM-11457](https://jira.camunda.com/browse/CAM-11457?focusedCommentId=155721&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-155721) for details.

File: multi-tenancy/schema-isolation/standalone.xml (hunk 3 above)
Comment (tasso94) [inline multi-tenancy/schema-isolation/standalone.xml:208]:
```suggestion
```

File: multi-tenancy/schema-isolation/standalone_test.xml (hunk 1 above)
Comment (tasso94) [inline multi-tenancy/schema-isolation/standalone_test.xml:208]:
```suggestion
```

**Review by tasso94 — APPROVED** [review]

**Pull request merged**